src/components/views/dashboard.tsx

```diff
@@ -39,6 +39,9 @@ export const Dashboard: FC<any> = ({navigation}) => {
   const exposure = useExposure();
   const {getNextScreen} = useSymptomChecker();
 
+  let isChartDataAvailable =
```

Comment:
```suggestion
  const isChartDataAvailable =
```
Doesn't need to be redefinable